assets/src/edit-story/components/library/panes/media/common/paginatedMediaGallery.js

```diff
@@ -197,20 +172,7 @@ function PaginatedMediaGallery({
       >
         <MediaGalleryInnerContainer>{mediaGallery}</MediaGalleryInnerContainer>
       </MediaGalleryContainer>
-      {providerType === ProviderType.UNSPLASH && (
-        <a
-          href={
-            'https://unsplash.com?utm_source=web_stories_wordpress&utm_medium=referral'
-          }
-          target={'_blank'}
-          rel={'noreferrer'}
-        >
-          <AttributionPill>
-            {__('Powered by', 'web-stories')}
-            <UnsplashLogoFull style={LOGO_PROPS} />
-          </AttributionPill>
-        </a>
-      )}
+      {Providers[providerType].attributionBuilder()}
```

Comment:
To make it clear what 'attributionBuilder' is, I would suggest:
```
  const AttributionComponent = Providers[providerType].attributionBuilder;
  return (
    <>
      <MediaGalleryContainer
        data-testid="media-gallery-container"
        ref={refCallbackContainer}
      >
        <MediaGalleryInnerContainer>{mediaGallery}</MediaGalleryInnerContainer>
      </MediaGalleryContainer>
      <AttributionComponent />
    </>
  );
```

Reply:
I renamed the field to attributionComponent so it should be clear now.